src/autogluon/bench/runbenchmark.py

```diff
@@ -130,6 +126,38 @@ def run_benchmark(
     _dump_configs(benchmark_dir=benchmark.metrics_dir, configs=configs, file_name="configs.yaml")
 
     if configs.get("METRICS_BUCKET", None):
+        if ":" in module_kwargs["run_kwargs"]["framework"]:
+            framework_name = module_kwargs["run_kwargs"]["framework"].split(":")[0]
+        else:
+            framework_name = module_kwargs["run_kwargs"]["framework"]
+
+        upload_path = []
+        if module_name == "multimodal":
+            upload_path = [
+                framework_name,
+                module_kwargs["run_kwargs"].get(["dataset_name"], None),
+                module_kwargs["run_kwargs"].get(["constraint"], None),
+                configs.get("mode", None),
+            ]
+        elif module_name == "tabular":
```

Comment:
`.1h8c.local` should be before `Australian`. `Australian` and `5` (fold) should be adjacent to each-other in the name.

Reply:
This is what it is when using AMLB:

```
aws.tabular_full.1h8c.australian.5.autogluon_bq/
```

I think this is a nice ordering.

`aws` can potentially be dropped if it is redundant.